src/modm/platform/core/hosted/module.lb

```diff
@@ -47,11 +47,11 @@ def build(env):
         if not env.has_module(":test:architecture"):
             env.template("clock.cpp.in")
 
-    if target.family == "windows":
-        env.log.error("Assertions are not implemented!")
-        if env.has_module(":architecture:delay"):
-            env.log.error("Delay functions are not implemented!")
-    else:
+    # if target.family == "windows":
+    #     env.log.error("Assertions are not implemented!")
```

Comment:
I think we should at least test what assertions do to make sure they don't misbehave as it is right now. It also probably wouldn't be hard to implement a "print and bail" version of the assertion, if its current behavior is broken.

Reply:
I tested assertions, they seem to work